### PR TITLE
Migrate to Vite

### PR DESCRIPTION
**Pull request by MiroMargineanu**

### CODE REVIEW

File: src/components/Heading/components/Identity/Identity.tsx [hunk 1]
```diff
@@ -182,18 +171,23 @@ export const Identity = () => {
                 <input
                   type='text'
                   name={field.name}
-                  value={values[field.name]}
+                  value={(values as any)[field.name]}
```
Comment (CiprianDraghici) [inline src/components/Heading/components/Identity/Identity.tsx:174]: Why do you need to cast to any? I think the values should have a strict type

File: src/components/Heading/components/Identity/Identity.tsx [hunk 2]
```diff
@@ -182,18 +171,23 @@ export const Identity = () => {
...
                   autoComplete='off'
                   onChange={handleChange}
                   onBlur={handleBlur}
                   className={classNames(styles.input, {
-                    [styles.invalid]: errors[field.name] && touched[field.name]
+                    [styles.invalid]:
+                      (errors as any)[field.name] &&
```
Comment (CiprianDraghici) [inline src/components/Heading/components/Identity/Identity.tsx:180]: Also here

File: src/components/Heading/components/Identity/Identity.tsx [hunk 3]
```diff
@@ -182,18 +171,23 @@ export const Identity = () => {
...
                   })}
                 />
 
-                {errors[field.name] && touched[field.name] && (
-                  <span className={styles.error}>{errors[field.name]}</span>
-                )}
+                {(errors as any)[field.name] &&
+                  (touched as any)[field.name] && (
+                    <span className={styles.error}>
+                      {(errors as any)[field.name]}
```
Comment (CiprianDraghici) [inline src/components/Heading/components/Identity/Identity.tsx:188]: also here

File: src/components/Heading/components/Identity/Identity.tsx (hunk 2 above)
Comment (MiroMargineanu) [reply src/components/Heading/components/Identity/Identity.tsx]: Updated. 👍

File: src/components/Heading/components/Identity/Identity.tsx (hunk 1 above)
Comment (MiroMargineanu) [reply src/components/Heading/components/Identity/Identity.tsx]: Updated. 👍

File: src/components/Heading/components/Identity/Identity.tsx (hunk 3 above)
Comment (MiroMargineanu) [reply src/components/Heading/components/Identity/Identity.tsx]: Updated. 👍

File: src/components/Heading/components/Identity/Identity.tsx [hunk 4]
```diff
@@ -173,30 +162,39 @@ export const Identity = () => {
...
-                  className={classNames(styles.input, {
-                    [styles.invalid]: errors[field.name] && touched[field.name]
-                  })}
-                />
-
-                {errors[field.name] && touched[field.name] && (
-                  <span className={styles.error}>{errors[field.name]}</span>
-                )}
+          {fields.map((field: FieldType) => {
+            const currentValues = values as PayloadType;
```
Comment (CiprianDraghici) [inline src/components/Heading/components/Identity/Identity.tsx:168]: No need for cast. The type is inferred. Probably you didn't set the initialValues type correctly

Comment (MiroMargineanu) [reply src/components/Heading/components/Identity/Identity.tsx]: Fixed again. 👍

**Review by CiprianDraghici — APPROVED** [review]

**Pull request merged**